experimental/oak_baremetal_app_qemu/Cargo.toml

```diff
@@ -6,14 +6,15 @@ edition = "2021"
 license = "Apache-2.0"
 
 [features]
-default = ["pvh"]
+default = ["pvh", "virtio_console_channel"]
 pvh = ["rust-hypervisor-firmware-boot/pvh"]
 multiboot = []
-serial_channel = ["oak_baremetal_kernel/serial_channel"]
+simple_io_channel = ["oak_baremetal_kernel/simple_io_channel"]
+virtio_console_channel = ["oak_baremetal_kernel/virtio_console_channel"]
 
 [dependencies]
 bitflags = "*"
-oak_baremetal_kernel = { path = "../../experimental/oak_baremetal_kernel" }
+oak_baremetal_kernel = { path = "../../experimental/oak_baremetal_kernel", default-features = false }
```

Comment:
Why disable the default features, if you're just enabling `virtio_console_channel` again? (Or maybe it'd make sense to remove the `virtio_console_channel` from default features in the kernel crate?)

Reply:
Removed the default feature from the kernel crate.

Reply:
I had to add a default channel back. Cargo udeps does not check with --al-features, and the kernel does not build if we do not specify at least one channel. For now I switched to using the vsock channel as the default.